### ISSUE TITLE
ExecuteCommand stdout truncated with big output on node 10

### ISSUE THREAD

**Issue by agix** [n8n-io/n8n#352]
**Describe the bug**
Stdout of executed command is truncated according to the default `maxBuffer` options of `child_process` which is __200 * 1024__ in node < v12

**To Reproduce**
Execute the following workflow on 0.55.1-ubuntu :

```
{
  "nodes": [
    {
      "parameters": {
        "command": "/bin/bash -c \"printf 'a%.0s' {1..204801}\""
      },
      "name": "Execute Command",
      "type": "n8n-nodes-base.executeCommand",
      "typeVersion": 1,
      "position": [
        550,
        570
      ]
    },
    {
      "parameters": {
        "functionCode": "item.stdoutLength = item.stdout.length;\ndelete item.stdout;\nreturn item;"
      },
      "name": "FunctionItem",
      "type": "n8n-nodes-base.functionItem",
      "typeVersion": 1,
      "position": [
        760,
        570
      ]
    }
  ],
  "connections": {
    "Execute Command": {
      "main": [
        [
          {
            "node": "FunctionItem",
            "type": "main",
            "index": 0
          }
        ]
      ]
    }
  }
}
```

**Expected behavior**
`{"stdoutLength": 204801}`

**Environment (please complete the following information):**
0.55.1-ubuntu

**Additional context**
Either bump the nodejs version of docker-ubuntu to node >= v12 as they increased the default `maxBuffer` to __1024 * 1024__ and/or allow exec options to be passed into ExecuteCommand node type.

**Comment by janober** [comment]
Thanks a lot @agix !

The regular docker-image uses already Node v12. So the Ubuntu image should actually do the same. So changed the image accordingly.

The next version of n8n will automatically build the image with v12.

**Comment by agix** [comment]
perfect

**Comment by janober** [comment]
Got released with n8n@0.56.0
